nginx.conf

```diff
@@ -1,10 +1,21 @@
-worker_processes      1;
+worker_processes      auto;
+
+error_log  /var/log/nginx/error.log notice;
+pid        /tmp/nginx.pid;
 
 events {
   worker_connections  1024;
 }
 
 http {
+  proxy_temp_path /tmp/proxy_temp;
```

Comment:
I assume these changes are all necessary to be able to run as a non root, unprivileged user, right?